### PR TITLE
feat: ui cleaning [WEB-1499]

### PR DESCRIPTION
**Pull request by pentcle**
please see description and screenshots here:

https://linear.app/yearn/issue/WEB-1499/feat-ui-cleaning

### CODE REVIEW

File: src/client/components/app/Navigation/NavSidebar.tsx [hunk 1]
```diff
@@ -78,6 +89,14 @@ const RouterLink = styled(Link)<{ selected: boolean }>`
       color: ${props.theme.colors.primary};
       font-weight: 700;
     }
+    &:hover {
+        ${LinkIcon} {
+      fill: ${props.theme.colors.primary};
```
Comment (turtlemoji) [inline src/client/components/app/Navigation/NavSidebar.tsx:94]: bad indent

File: README.md [hunk 2]
```diff
@@ -34,6 +34,17 @@ Code style follows prettier conventions (`yarn prettier`). Commit messages follo
...
 
+### @yearn/web-lib access
+
+You will need to generate a [Github Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token), and create or update the ~/.npmrc file with the following content:
+
+```
+always-auth=true
+@yearn:registry=https://npm.pkg.github.com/
+registry=https://registry.npmjs.org
+//npm.pkg.github.com/:_authToken=YOUR_ACCESS_TOKEN
```
Comment (xgambitox) [inline README.md:45]:
```suggestion
//npm.pkg.github.com/:_authToken=YOUR_ACCESS_TOKEN
```
Since we are updating the global `.npmrc` file located in home, and not the one inside the project, we only need the suggested line

File: src/client/components/app/Navigation/NavSidebar.tsx [hunk 3]
```diff
@@ -6,7 +6,7 @@ import { SettingsSelectors } from '@store';
 import { NavigationLink } from '@components/app';
 import { Link, Icon, Logo, RedirectIcon } from '@components/common';
 
-const linkHoverFilter = 'brightness(90%)';
+// const linkHoverFilter = 'brightness(90%)';
```
Comment (xgambitox) [inline src/client/components/app/Navigation/NavSidebar.tsx:9]: If its no longer needed, lets remove it better :)

File: src/client/components/app/VaultDetail/VaultDetailPanels.tsx [hunk 4]
```diff
@@ -108,7 +109,7 @@ const OverviewInfo = styled(Card)`
   a {
     text-decoration: underline;
     color: inherit;
-    color: ${({ theme }) => theme.colors.titles};
+    //color: ${({ theme }) => theme.colors.titles};
```
Comment (xgambitox) [inline src/client/components/app/VaultDetail/VaultDetailPanels.tsx:112]: If its no longer needed, lets remove it better :)

File: src/client/components/app/Navigation/NavSidebar.tsx (hunk 1 above)
Comment (pentcle) [reply src/client/components/app/Navigation/NavSidebar.tsx:94]: my auto prettier told me to do it that way - how should it be?

File: src/client/components/app/VaultDetail/VaultDetailPanels.tsx (hunk 4 above)
Comment (pentcle) [reply src/client/components/app/VaultDetail/VaultDetailPanels.tsx:112]: yes great ty

File: src/client/components/app/Navigation/NavSidebar.tsx (hunk 3 above)
Comment (pentcle) [reply src/client/components/app/Navigation/NavSidebar.tsx:9]: ty

**Review by xgambitox — APPROVED** [review]

File: src/client/components/app/Navbar/index.tsx [hunk 5]
```diff
@@ -36,7 +36,7 @@ const StyledText = styled.h1<{ toneDown?: boolean }>`
   ${({ toneDown, theme }) =>
     toneDown &&
     `
-    color: ${theme.colors.texts};
+    color: ${theme.colors.textsVariant};
```
Comment (turtlemoji) [inline src/client/components/app/Navbar/index.tsx:39]: Dark before:
image

Dark now:
image

Is this change intended?

File: src/client/components/app/Navigation/NavSidebar.tsx [hunk 6]
```diff
@@ -6,7 +6,6 @@ import { SettingsSelectors } from '@store';
 import { NavigationLink } from '@components/app';
 import { Link, Icon, Logo, RedirectIcon } from '@components/common';
 
-const linkHoverFilter = 'brightness(90%)';
```
Comment (turtlemoji) [inline src/client/components/app/Navigation/NavSidebar.tsx:9]: this removes hover color change in the themes: `cyberpunk, classic, explorer`.

File: src/client/components/app/SliderCard/index.tsx [hunk 7]
```diff
@@ -44,7 +44,16 @@ const StyledCardContent = styled(CardContent)`
   margin-bottom: 0;
   font-size: 1.6rem;
   line-height: 2.4rem;
+  padding-right: 12.8rem;
```
Comment (turtlemoji) [inline src/client/components/app/SliderCard/index.tsx:47]: I would remove this paddings (tablet and mobile too), looks weird. Maybe apply them in the custom banner when it is necessary.

image
image

File: src/client/components/app/Navigation/NavSidebar.tsx (hunk 6 above)
Comment (turtlemoji) [reply src/client/components/app/Navigation/NavSidebar.tsx:9]: Also sidebar default color seems wrong on classic:
image

File: src/client/components/app/VaultDetail/VaultDetailPanels.tsx [hunk 8]
```diff
@@ -53,6 +53,7 @@ const VaultChart = styled(Card)`
 
 const StyledCardContent = styled(CardContent)`
   margin-top: 0.4rem;
+  color: ${({ theme }) => theme.colors.texts};
```
Comment (turtlemoji) [inline src/client/components/app/VaultDetail/VaultDetailPanels.tsx:56]: wrong color on classic:
Before:
image


Now:
image

Comment (turtlemoji) [reply src/client/components/app/VaultDetail/VaultDetailPanels.tsx:56]: Looks better than before on cyberpunk tho!

File: src/client/components/app/VaultDetail/VaultDetailPanels.tsx [hunk 9]
```diff
@@ -245,7 +245,7 @@ const VaultOverview = styled(Card)`
 const StyledChartTab = styled.span<{ active?: boolean }>`
   margin: 10px;
   cursor: pointer;
-  color: ${(props) => (props.active ? props.theme.colors.secondary : props.theme.colors.primaryVariant)};
+  color: ${(props) => (props.active ? props.theme.colors.primary : props.theme.colors.texts)};
```
Comment (turtlemoji) [inline src/client/components/app/VaultDetail/VaultDetailPanels.tsx:248]: nice!

File: src/client/routes/Disclaimer/index.tsx [hunk 10]
```diff
@@ -1,16 +1,47 @@
 import styled from 'styled-components';
```
Comment (turtlemoji) [inline src/client/routes/Disclaimer/index.tsx:2]: Much better disclaimer nice, let me know if we need to translate it to other locales or just EN is ok

File: src/client/components/app/Navbar/index.tsx (hunk 5 above)
Comment (pentcle) [reply src/client/components/app/Navbar/index.tsx:39]: no - will fix ty

File: src/client/routes/Disclaimer/index.tsx (hunk 10 above)
Comment (pentcle) [reply src/client/routes/Disclaimer/index.tsx:2]: oh yes this will need translating

File: src/client/components/app/SliderCard/index.tsx (hunk 7 above)
Comment (pentcle) [reply src/client/components/app/SliderCard/index.tsx:47]: these need to be there for the default wide banners as the text should not go full width

image

 - we are changing to the web-lib banners next week so this is prob ok to proceed with for now
 - the above look ok to me? and it fixes these: 
image

File: src/client/components/app/VaultDetail/VaultDetailPanels.tsx (hunk 8 above)
Comment (pentcle) [reply src/client/components/app/VaultDetail/VaultDetailPanels.tsx:56]: figma file specifies the darker grey - we have agreed with themes beyond light and dark we are ok to not comply with accessibility rules: 

image

File: src/client/components/app/Navigation/NavSidebar.tsx (hunk 6 above)
Comment (pentcle) [reply src/client/components/app/Navigation/NavSidebar.tsx:9]: classic is as per figma. i did notice Vaults LH is not active when user is in Vaults / VaultName

image

Ideally Vaults LH nav should remain active when anywhere in that section - i'll see if i can fix that

Comment (pentcle) [reply src/client/components/app/Navigation/NavSidebar.tsx:9]: hover is now specific colour

image

Comment (pentcle) [reply src/client/components/app/Navigation/NavSidebar.tsx:9]: > Also sidebar default color seems wrong on classic: image

good catch - need to get figma aligned with codebase colour naming patterns (ie by object name, not colour name)

image

File: src/client/components/app/Navbar/index.tsx (hunk 5 above)
Comment (pentcle) [reply src/client/components/app/Navbar/index.tsx:39]: image

**Comment by pentcle** [comment]
thank you - amazing to have sets of extra eyes to spot things

**Comment by pentcle** [comment]
added link to YFI Buyback 
image

File: src/client/components/app/SliderCard/index.tsx (hunk 7 above)
Comment (turtlemoji) [reply src/client/components/app/SliderCard/index.tsx:47]: okay.

**Comment by turtlemoji** [comment]
> added link to YFI Buyback image

image

We should modify the media query so it changes to this faster then:
image

**Comment by pentcle** [comment]
> > added link to YFI Buyback image
> 
> image
> 
> We should modify the media query so it changes to this faster then: image

changed to desktopS (tabletL didn't break it soon enough)
image

File: src/client/components/app/Navigation/NavSidebar.tsx (hunk 6 above)
Comment (pentcle) [reply src/client/components/app/Navigation/NavSidebar.tsx:9]: image

image

image

**Review by FoxTheSin — APPROVED** [review]

**Review by xgambitox — APPROVED** [review]
